app/Http/Controllers/BlogController.php

```diff
@@ -123,6 +123,6 @@ public function delete($slug)
         $blog->delete();
 
         return redirect('/blog')
-            ->with('message', 'Your post has been deleted!');
+            ->with('message', 'Your post has been deleted successfully!');
```

Comment:
Also make the "successfully" "Successfully"

Reply:
Change the "successfully" in the success message in BlogController to "Successfully"